app/src/main/java/org/mozilla/focus/utils/Debounce.kt

```diff
@@ -22,8 +21,7 @@ import kotlinx.coroutines.launch
  * @param channel the channel to debounce
  * @return a throttled channel
  */
-@ObsoleteCoroutinesApi
-@ExperimentalCoroutinesApi
+@OptIn(ExperimentalCoroutinesApi::class)
```

Comment:
Using `@OptIn(ExperimentalCoroutinesApi::class)` here instead of `@ExperimentalCoroutinesApi` so that you don't have to propagate it in the entire call chain.